aiconfig-docs/docs/getting-started.md

```diff
@@ -58,29 +54,82 @@ $ yarn add aiconfig
 
 </Tabs>
 
-:::caution
-Make sure to specify the API keys (such as `OPENAI_API_KEY`) in your environment before proceeding.
-:::
+**Note:** You need to install the python AIConfig package to create and edit your configs using the AIConfig Editor. You can still use the AIConfig Node SDK to interact with your config in your application code.
+
+## Setup your OpenAI API Key
 
-## Quickstart
+For this tutorial, you will need to have an OpenAI key that has access to GPT-4.
 
-In this quickstart, you will create a customizable NYC travel itinerary using `aiconfig`. We will start with a pre-built AIConfig that we generated from this [AI Workbook](https://lastmileai.dev/workbooks/clooqs3p200kkpe53u6n2rhr9).
+1. Get your OpenAI API Key: https://platform.openai.com/account/api-keys
+2. Open Terminal
+3. Add this line, replace ‘your-api-key-here’ with your API key: `export OPENAI_API_KEY='your-api-key-here'`
 
-:::tip
-Clone the tutorial code used below from [**here**](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/Getting-Started)
-:::
+## Open AIConfig Editor
 
-<p align="center">
-<video controls height="480" width="800">
-    <source src="https://github.com/lastmile-ai/aiconfig/assets/25641935/d3d41ad2-ab66-4eb6-9deb-012ca283ff81"/>
-  </video>
-</p>
+AIConfig Editor allows you to visually create and edit the prompt chains and model parameters that are stored as AIConfigs. You can also chain prompts and use global and local variables in your prompts.
 
-### 1. Download the AIConfig.
+1. Open your Terminal
+2. Run this command: `aiconfig edit --aiconfig-path travel.aiconfig.json`
 
-This AIConfig `travel.aiconfig.json` contains a prompt chain to get a list of travel activities from an LLM and then customize the activities based on user preferences (defined as parameters of the prompt). It also contains the specific models and model parameters for the LLMs.
+This will open AIConfig Editor in your default browser and create a new AIConfig JSON file `travel.aiconfig.json` in your current directory.
```

Comment:
Should probably say at `http://localhost:8080/`

Reply:
Sg!